<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-M1R1IP0R" failures="3" tests="10" name="Tek Insurance API Tests" time="15.861" errors="0" timestamp="2023-06-03T11:03:57 EDT">
  <testcase classname="tek.api.sqa.tests.AddPrimaryAccountTest" name="addPrimaryAccount" time="2.617">
    <failure type="java.lang.AssertionError" message="expected [201] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [400]
at tek.api.sqa.tests.AddPrimaryAccountTest.addPrimaryAccount(AddPrimaryAccountTest.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- addPrimaryAccount -->
  <testcase classname="tek.api.sqa.tests.CreateAccountTest" name="createAccountValidTest" time="0.665">
    <failure type="java.lang.RuntimeException" message="Date parse exception">
      <![CDATA[java.lang.RuntimeException: Date parse exception
at tek.api.utility.DataGenerator.getDateOfBirth(DataGenerator.java:48)
at tek.api.sqa.tests.CreateAccountTest.createAccountValidTest(CreateAccountTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- createAccountValidTest -->
  <testcase classname="tek.api.sqa.tests.GetAccountTest" name="getAccountWithAnID" time="1.251"/>
  <testcase classname="tek.api.sqa.tests.GetAllAccountsTest" name="getAllAccounts" time="5.977"/>
  <testcase classname="tek.api.sqa.tests.GetAllPlanCodeTest" name="getAllPlanCode" time="1.188"/>
  <testcase classname="tek.api.sqa.tests.SecurityTest" name="testGeneratedTokenWithInvalidUsername" time="0.455"/>
  <testcase classname="tek.api.sqa.tests.SecurityTest" name="testGeneratedTokenWithValidUsername" time="0.595"/>
  <testcase classname="tek.api.sqa.tests.TokenVerifyTest" name="tokenVerifyTest" time="0.939"/>
  <testcase classname="tek.api.sqa.tests.TokenVerifyTest" name="verifyTestWithInvalidToken" time="0.369"/>
  <testcase classname="tek.api.sqa.tests.UserProfileTest" name="userProfileTest" time="1.028">
    <failure type="java.lang.AssertionError" message="expected [200] but found [403]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [403]
at tek.api.sqa.tests.UserProfileTest.userProfileTest(UserProfileTest.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- userProfileTest -->
</testsuite> <!-- Tek Insurance API Tests -->
